### PR TITLE
[SLC-2020] Opening CFP and updating associated dates

### PR DESCRIPTION
**Pull request by Clait**
-Open CFP
-Update dates and text around CFP

### CODE REVIEW

File: data/events/2020-salt-lake-city.yml [hunk 1]
```diff
@@ -12,15 +12,15 @@ ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it
...
 startdate: 2020-06-02T08:00:00-06:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate: 2020-06-03T18:00:00-06:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2020-06-03T18:00:00-06:00   # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start: 2019-11-15T07:01:00-06:00  # start accepting talk proposals.
+cfp_date_start: 2019-08-30T07:01:00-06:00  # start accepting talk proposals.
 cfp_date_end: 2020-01-07T07:01:00-06:00 # close your call for proposals.
-cfp_date_announce:  2019-11-15T07:01:00-06:00 # inform proposers of status
+cfp_date_announce:  2019-08-30T07:01:00-06:00 # inform proposers of status
```
Comment (bridgetkromhout) [inline data/events/2020-salt-lake-city.yml]: I understand that "announce" is confusing wording. This is supposed to be the date you intend to let people know whether or not they'll be speaking.

**Comment by Clait** [comment]
Ah, ok.. will update

**Comment by Clait** [comment]
cfp_announce updated.

File: data/events/2020-salt-lake-city.yml (hunk 1 above)
Comment (Clait) [reply data/events/2020-salt-lake-city.yml]: Revised, thanks

**Comment by Clait** [comment]
Thanks

**Pull request merged**